src/modules/chat_font_settings/index.js

```diff
@@ -12,9 +12,17 @@ Leave the field blank to use the default.`;
 
 const STYLE_ID = 'bttv-font-size';
 
+const encodeFontFamily(fontFamily) => {
+    if (['serif', 'sans-serif', 'monospace', 'cursive', 'fantasy', 'system-ui'].indexOf(fontFamily)) {
+        return html.escape(fontFamily);
+    } else {
+        return '"' + html.escape(fontFamily) + '"';
```

Comment:
you should use template literals

Reply:
the logic can also be improved by always escaping the passed font family and then choosing to wrap it in quotes if it is not a generic name.